lib/kubernetes-deploy/kubernetes_resource/custom_resource.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+require 'jsonpath'
+module KubernetesDeploy
+  class CustomResource < KubernetesResource
+    def initialize(namespace:, context:, definition:, logger:, statsd_tags: [], crd:)
+      super(namespace: namespace, context: context, definition: definition,
+        logger: logger, statsd_tags: statsd_tags)
+      @crd = crd
+    end
+
+    def timeout
```

Comment:
How is this different from the super class behavior?

Reply:
Ah, it isn't. Leftover from my attempts at adding some more configurability (decided it's outside of this PR's scope)